arbor/CMakeLists.txt

```diff
@@ -137,7 +137,7 @@ if(ARB_WITH_CUDA_CLANG)
     target_link_libraries(arbor PUBLIC  ${CUDA_LIBRARIES})
 endif()
 
-set_target_properties(arbor PROPERTIES CUDA_RESOLVE_DEVICE_SYMBOLS ON)
+set_target_properties(arbor PROPERTIES CUDA_RESOLVE_DEVICE_SYMBOLS ON POSITION_INDEPENDENT_CODE ON)
```

Comment:
There's already this line in the main `CMakeLists.txt`: https://github.com/arbor-sim/arbor/blob/master/CMakeLists.txt#L244
```
set(CMAKE_POSITION_INDEPENDENT_CODE ON)
```
So I think this may not be needed?

Reply:
Good point!
Any ideas about the reported relocation errors we're seeing in the Travis logs?

Reply:
I was able to successfully build pyarb on my system (ubuntu 20.04, gcc 9.3) with the same cmake flags that cause the error on travis. 
I can try again with gcc 8.4 as soon as it's done building. 
Could it be the order of the flag?